docs/reference/form_help_message.rst

```diff
@@ -130,34 +130,6 @@ This Extension for example adds a note field to some entities which use a custom
         }
     }
 
-Help messages in a sub-field
```

Comment:
Should we deprecat `addHelp` too?

Reply:
I've never used that method, let me take a look